EIPS/eip-X.md

```diff
@@ -0,0 +1,198 @@
+
+To provide more scale to Ethereum, while not inducing a restrictively high burden on both consumer and consensus nodes, eth2 introduces a "sharded" solution in which a number of blockchain shards -- each of similar capacity to Ethereum mainnet today -- run in parallel under a unified consensus mechanism. The core consensus (the beacon chain) and small number of these shards can be processed via a single consumer machine, while the aggregate of the system provides much higher capacity.
+
+See the [Ethereum wiki sharding FAQ](https://eth.wiki/sharding/Sharding-FAQs) for an excellent introduction and discussion of scaling through sharding.
+
+### Decentralization and economic finality through proof-of-stake
+
+Since the [early days](https://blog.ethereum.org/2015/12/28/understanding-serenity-part-2-casper/) of Ethereum, proof-of-stake has been a long-awaited promise of the following:
+* Increased decentralization of the core consensus by lowering the barrier to entry and technical requirements of participation 
+* Increased cryptoeconomic security via in-protocol penalties for misbehaviour and the addition of economic finality
+* Elimination of the energy hungry mining of the current PoW consensus mechanism
+
+In addition to the above, PoS has synergies with the sharding scaling solution. Due to the random sampling requirement of sharding, PoS provides a more simple and direct access to the ["active validator set"](https://eth.wiki/sharding/Sharding-FAQs#how-do-you-actually-do-this-sampling-in-proof-of-work-and-in-proof-of-stake) than PoW and thus allows for a more direct sharded protocol construction.
+
+See the [Ethereum wiki proof-of-stake FAQ](https://eth.wiki/en/concepts/proof-of-stake-faqs) for an excellent introduction and discussion of proof-of-stake consensus.
+
+## Parameters
+
+| Parameter | Value |
+| - | - |
+| `SPEC_RELEASE_VERSION` | TBD |
+| `SPEC_RELEASE_COMMIT` | TBD |
+| `DEPOSIT_CONTRACT_ADDRESS` | TBD |
+| `MIN_GENESIS_TIME` | TBD |
+| `BASE_REWARD_FACTOR` | `2**6` (64) |
+| `INACTIVITY_PENALTY_QUOTIENT` | `2**26` (67,108,864) |
+| `PROPORTIONAL_SLASHING_MULTIPLIER` | `1` |
+| `MIN_SLASHING_PENALTY_QUOTIENT` | `2**5` (128) | 
+
+_Note:_ Eth2 has many more [Phase 0 configuration parameters](https://github.com/ethereum/eth2.0-specs/blob/dev/configs/mainnet/phase0.yaml) but the majority are left out of this EIP for brevity.
+
+## Specification
+
+Phase 0 is designed to require _no breaking consensus changes_ to existing Ethereum mainnet. Instead, this is an extension of Ethereum mainnet to bootstrap the new PoS consensus.
+
+Phase 0 specifications are maintained in a [repository](https://github.com/ethereum/eth2.0-specs) independent of this EIP. [`SPEC_RELEASE_VERSION` release]() of the specs at `SPEC_RELEASE_COMMIT` are considered the canonical Phase 0 specs for this EIP.
```

Comment:
If the specification lives elsewhere, why bother with an EIP?  This feels largely redundant to create an EIP only to point it somewhere else.  It would be like having an EIP that just points at an RFC.  At best you are wasting effort duplicating content, at worst the two end up out of sync.